habitat_baselines/evaluate_ppo.py

```diff
@@ -112,8 +188,17 @@ def main():
         args.num_processes, args.hidden_size, device=device
     )
     not_done_masks = torch.zeros(args.num_processes, 1, device=device)
+    stats_episodes = set()
+    if args.video == 1:
+        rgb_frames = [[]] * args.num_processes
+        if not os.path.exists(args.out_dir_video):
+            os.makedirs(args.out_dir_video)
+    else:
```

Comment:
NIT: Set rgb_frames to None first and then jump into if clause, this will save one line.